carChangers.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+var CarLot = (function(originalCarLot) {
+
+  // Removes targeted features from car elements
+  originalCarLot.removeTargeting = function() {
+    // Loops through car elements removing targeted class and resetting borders
+    for (let i = 0; i < carCard.length; i++) {
+      carCard[i].classList.remove("targeted");
+      carCard[i].setAttribute("style", "border: 2px solid " + inventory.cars[i].color);
+    }  
+    // Clears the input area, if need be, and places cursor there for immeadiate typing
+    textField.value = "";
+    textField.focus();
+  };
+
+  // Adds targeted styling to a specific car element and then calls the function to change the description
+  originalCarLot.addTargeting = function (clickTarget) {
```

Comment:
Take a look at this requirement: "The other function changes the thickness of the border of a car element, and changes its background color. The function must accept two arguments.
    A car DOM element that was clicked on.
    A color name."
You're most of the way there, but you do need to make this take a color as an argument. It doesn't have to be the border color, but try making it the background color.